src/main/java/frc/robot/commands/swervedrive/auto/AutoBalanceCommand.java

```diff
@@ -0,0 +1,86 @@
+package frc.robot.commands.swervedrive.auto;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.swervedrive.SwerveSubsystem;
+
+
+/**
+ * Auto Balance command using a simple PID controller. Created by Team 3512
+ * <a href="https://github.com/frc3512/Robot-2023/blob/main/src/main/java/frc3512/robot/commands/AutoBalance.java">...</a>
+ */
+public class AutoBalanceCommand extends Command
+{
+
+  private final SwerveSubsystem swerveSubsystem;
+  private final PIDController   controller;
+
+  public AutoBalanceCommand(SwerveSubsystem swerveSubsystem)
+  {
+    this.swerveSubsystem = swerveSubsystem;
+    controller = new PIDController(1.0, 0.0, 0.0);
+    controller.setTolerance(1);
+    controller.setSetpoint(0.0);
+    // each subsystem used by the command must be passed into the
+    // addRequirements() method (which takes a vararg of Subsystem)
+    addRequirements(this.swerveSubsystem);
+  }
+
+  /**
+   * The initial subroutine of a command.  Called once when the command is initially scheduled.
+   */
+  @Override
+  public void initialize()
+  {
+
+  }
+
+  /**
+   * The main body of a command.  Called repeatedly while the command is scheduled. (That is, it is called repeatedly
+   * until {@link #isFinished()}) returns true.)
+   */
+  @Override
+  public void execute()
+  {
+    SmartDashboard.putBoolean("At Tolerance", controller.atSetpoint());
+
+    double translationVal = MathUtil.clamp(controller.calculate(swerveSubsystem.getPitch().getDegrees(), 0.0), -0.5,
+                                           0.5);
```

Comment:
I get why this 0.5 is on a newline but it doesn't look great lol

Reply:
That is just vs code